package.json

```diff
@@ -48,6 +48,7 @@
     "ts-jest": "^25.2.0"
   },
   "scripts": {
+    "jsonlint": "node scripts/jsonlint.js",
```

Comment:
is this separate to `yarn lint` 
`yarn lint` should lint everything

Reply:
I'm not sure to understand the direction you want to go in. Are you mentioning:

- Eslint? It can be used for linting JSON? I didn't look into it.
- The script command? I don't think that it matters. It's not something that is meant to be part of the daily workflow, but instead, runs in the CI and to be easily debuggable.

But I could do: rename the current lint -> eslint. add a new lint script that batches them all. Would it work with you?

Reply:
> But I could do: rename the current "lint" script -> "eslint" and add a new "lint" script that batches all the lint commands. Would it work with you?

Yes perfect!
So yarn lint => lint everything (no brainer)

Reply:
Sounds like a plan